static/locales/en.json

```
@@ -109,12 +109,18 @@
         "live2dPreviewLoadFailed": "Live2D model preview loading failed: {{error}} ",
-            "vrmModuleLoadFailed": "Failed to load the VRM module",
-            "vrmPreviewLoaded": "VRM model preview loaded",
-            "vrmPreviewFailed": "Failed to load the VRM model preview",
-            "mmdModuleLoadFailed": "Failed to load the MMD module",
-            "mmdPreviewLoaded": "MMD model preview loaded",
-            "mmdPreviewFailed": "Failed to load the MMD model preview",
+        "coverBgBottom": "Bottom background layer",
+        "characterCardCover": "Character card cover",
+        "coverBgTop": "Top background layer",
+        "selectCharacterForCover": "Select a character to view the cover",
+        "searchAriaLabel": "Search subscriptions",
```

Biome (2.4.10)

[error] The key searchAriaLabel was already declared.

(lint/suspicious/noDuplicateObjectKeys)